cxx4opencl/address_spaces.txt

```diff
@@ -20,52 +20,20 @@ logic.
 
 ==== Casts
 
-C-style casts follow `OpenCL C v2.0 rules (s6.5.5)`. All cast operators
-permit conversion to generic address space implicitly. However converting
+C-style casts follow rules from `OpenCL C v2.0 s6.5.5`. All cast operators
+permit conversion to generic address space implicitly. However, converting
```

Comment:
, however,

Reply:
This reads slightly strangely. Could we say "conversions to generic address space can be made by any cast operator; converting from generic to named ..."